addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4818,7 +4818,8 @@ registry.sizing = SnippetOptionWidget.extend({
                             .elementsFromPoint(ev.pageX, ev.pageY);
                         const toBeClickedEl = sameCoordinatesEls
                             .find(el => !el.closest("#oe_manipulators"));
-                        if (toBeClickedEl) {
+                        // Check toBeClickEl has native JS `click` function
+                        if (toBeClickedEl && typeof toBeClickedEl.click === "function") {
```

Comment:
- I’d rather check this in the `find()` method, so the SVG is excluded from `sameCoordinatesEls` instead of being returned by `find()` and left unused. This way, the click can be triggered on the column behind it for instance.
```js
.find(el => !el.closest("#oe_manipulators") && typeof el.click === "function");
```
- I found another problem related to this: if you try to select a column by clicking on an `s_process_step_connector`, the column isn’t selected. I think we could add `pointer-events: none` to the connectors to improve this.

Reply:
Hello @bvr-odoo 

>I think we could add pointer-events: none

This disables all pointer events for the selected element, allowing interactions to pass through to elements underneath. This approach should eliminate the need for the JS code and solve our problem. What do you think?

Reply:
> Hello @bvr-odoo
> 
> > I think we could add pointer-events: none
> 
> This disables all pointer events for the selected element, allowing interactions to pass through to elements underneath. This approach should eliminate the need for the JS code and solve our problem. What do you think?

- I would leave the js, it could be useful for other use cases.
- Please review your commit message, there are typos.
**coloumns** → colonne
**Thisfix** → This fix
- Use the same text as the commit message for the PR description.